plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/PurRepository.java

```diff
@@ -343,11 +342,6 @@ public RootRef getRootRef() {
           log.logDetailed( BaseMessages.getString( PKG, "PurRepositoryMetastore.Create.Message" ) );
         }
         metaStore = new PurRepositoryMetaStore( this );
```

Comment:
If PurRepository creates an instance of PurRepositoryMetastore, how would it connect to VFS metastore

Reply:
This metaStore doesn't go anywhere. Everywhere else still uses the metaStore from MetastoreLocatorImpl.

Reply:
This metastore is used when connected to repository. Is this story handling the usecase of using VFS Metastore when connected to repository

Reply:
No, in this story repository still uses the repository metastore, just not hard-coded.